plop-templates/back-end/api/delete.hbs

```diff
@@ -1,6 +1,7 @@
 import { Delete{{pascalCase object}}Response } from "../../../types/openapi";
 import {
-  delete{{pascalCase object}},
```

Comment:
There was a conflict with the naming of this (e.g. `deleteMetric` from the model and `deleteMetric` from the controller handler name).